### PR TITLE
Update report scenarios

### PR DESCRIPTION
**Pull request by axthosarouris**
Update the report scenarios for readability

### CODE REVIEW

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinJournalPublicationJournal.java [hunk 1]
```diff
@@ -23,20 +23,19 @@
...
-    public static final String PUBLISHER_NAME = "tidsskriftnavn";
+    public static final String JOURNAL_TITLE = "tidsskriftnavn";
 
     @JsonProperty(ISSN)
     private String issn;
     @JsonProperty(ISSN_ONLINE)
     private String issnOnline;
-    @JsonProperty(PUBLISHER_NAME)
-    private String publisherName;
+    @JsonProperty(JOURNAL_TITLE)
```
Comment (axthosarouris) [inline cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinJournalPublicationJournal.java:32]: fix the naming to correspond to the correct meaning

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinBookOrReport.java [hunk 2]
```diff
@@ -20,18 +20,18 @@
...
-        "status_utgitt_av_forlag", "stedangivelse_utgiver", "landkode_utgiver", "institusjonsnr_utgiver",
-        "avdnr_utgiver", "undavdnr_utgiver", "gruppenr_utgiver", "tidsskriftnr_serie", "volum_serie",
-        "sprakkode_oversatt_fra", "sprakkode_oversatt_til", "originalforfatter", "originaltittel",
-        "forlag", "tidsskrift_serie", "arkivpost"})
-public class CristinBookReport {
+    "status_utgitt_av_forlag", "stedangivelse_utgiver", "landkode_utgiver", "institusjonsnr_utgiver",
+    "avdnr_utgiver", "undavdnr_utgiver", "gruppenr_utgiver", "tidsskriftnr_serie", "volum_serie",
+    "sprakkode_oversatt_fra", "sprakkode_oversatt_til", "originalforfatter", "originaltittel",
+    "forlag", "tidsskrift_serie", "arkivpost"})
+public class CristinBookOrReport {
```
Comment (axthosarouris) [inline cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinBookOrReport.java:27]: This class represents objects that appear under the name "type_bok_rapport" and contains information that is found in books and reports. Renamed for better understanding

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinBookOrReportMetadata.java [hunk 3]
```diff
@@ -20,18 +20,18 @@
...
-        "status_utgitt_av_forlag", "stedangivelse_utgiver", "landkode_utgiver", "institusjonsnr_utgiver",
-        "avdnr_utgiver", "undavdnr_utgiver", "gruppenr_utgiver", "tidsskriftnr_serie", "volum_serie",
-        "sprakkode_oversatt_fra", "sprakkode_oversatt_til", "originalforfatter", "originaltittel",
-        "forlag", "tidsskrift_serie", "arkivpost"})
-public class CristinBookReport {
+    "status_utgitt_av_forlag", "stedangivelse_utgiver", "landkode_utgiver", "institusjonsnr_utgiver",
+    "avdnr_utgiver", "undavdnr_utgiver", "gruppenr_utgiver", "tidsskriftnr_serie", "volum_serie",
+    "sprakkode_oversatt_fra", "sprakkode_oversatt_til", "originalforfatter", "originaltittel",
+    "forlag", "tidsskrift_serie", "arkivpost"})
+public class CristinBookOrReportMetadata {
```
Comment (axthosarouris) [inline cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinBookOrReportMetadata.java:27]: This class represents objects that appear under the name "type_bok_rapport" and contains information that is found in books and reports. Renamed for better understanding

File: cristin-import/src/test/resources/features/ChapterArticleRules.feature [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
-  ChapterArticle when the Cristin Result's secondary category is "Kapittel"
+  Scenario: Cristin Result of type "Chapter" maps to NVA entry of type
+  "ChapterArticle"
     Given a valid Cristin Result with secondary category "FAGLIG_KAPITTEL"
     When the Cristin Result is converted to an NVA Resource
     Then the NVA Resource has a Publication Instance of type "ChapterArticle"
 
-  Scenario: Map returns NVA Resource with Reference having a PublicationInstance of type
-  ChapterArticle when the Cristin Result's secondary category is "Kapittel"
+  Scenario: MCristin Result of type "Popular scientific chapter/article" maps to NVA entry of type
```
Comment (Eirik-Eide-Saur) [inline cristin-import/src/test/resources/features/ChapterArticleRules.feature:15]: Typo "MCristin"

File: cristin-import/src/test/resources/features/ChapterArticleRules.feature [hunk 5]
```diff
@@ -1,19 +1,19 @@
 Feature:
 
-  Scenario: Map returns NVA Resource with Reference having a PublicationInstance of type
-  ChapterArticle when the Cristin Result's secondary category is "Kapittel"
+  Scenario: Cristin Result of type "Academic chapter/article/Conference paper" maps to NVA entry of type
+  "ChapterArticle"
```
Comment (Eirik-Eide-Saur) [inline cristin-import/src/test/resources/features/ChapterArticleRules.feature:4]: I believe this scenario description gives a false impression of what the test actually proves. It claims that "Academic chapter/article/Conference paper" gets mapped, but in reality it only proves this for chapter

Comment (axthosarouris) [reply cristin-import/src/test/resources/features/ChapterArticleRules.feature:4]: Academic chapter/article/Conference is the English description of the type Kapittel accroding to the excel file in Confluence

Comment (Eirik-Eide-Saur) [reply cristin-import/src/test/resources/features/ChapterArticleRules.feature:4]: Aaaah, my bad

**Review by Eirik-Eide-Saur — APPROVED** [review]

**Pull request merged**